flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/stream/sql/join/JoinReorderITCase.java

```diff
@@ -42,6 +47,20 @@ public class JoinReorderITCase extends JoinReorderITCaseBase {
 
     private StreamExecutionEnvironment env;
 
+    @BeforeEach
+    public void before() throws Exception {
+        super.before();
+        // This conf is aims to fix IOException due to timeout occurring while requesting exclusive
+        // NetworkBuffer (see https://issues.apache.org/jira/browse/FLINK-30727).
+        tEnv.getConfig().getConfiguration().set(TaskManagerOptions.NETWORK_MEMORY_FRACTION, 0.4f);
```

Comment:
make sense to me this fix can make the test stable, but eventually we should remove this config